schemas/bridging/getAppMetadataBridgeResponse.schema.json

```diff
@@ -10,19 +10,8 @@
     {
```

Comment:
Need to reference in the agentResponse schema here - I also missed this in a few early ones